presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimate.java

```diff
@@ -37,11 +37,16 @@
 public class PlanNodeStatsEstimate
 {
     private static final double DEFAULT_DATA_SIZE_PER_COLUMN = 50;
-    public static final PlanNodeStatsEstimate UNKNOWN_STATS = builder().build();
```

Comment:
What is the difference?

Reply:
Yes and no -- one could argue that builder, being `PlanNodeStatsEstimate`'s member, is known to `PlanNodeStatsEstimate` and so we can assume certain builder's behavioral properties here.

Reply:
When there is a change to the builder defaults, you may not even notice it was used to define the UNKNOWN constant. Although the change to default may seem logical, it might not be in terms of UNKNOWN. So, potentially it is more error prone.

Reply:
Inconceivable to me :)